src/commands/infra/scaffold.test.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  disableVerboseLogging,
+  enableVerboseLogging,
+  logger
+} from "../../logger";
+import { generateClusterDefinition, parseVariablesTf } from "./scaffold";
+
+beforeAll(() => {
+  enableVerboseLogging();
+  jest.setTimeout(30000);
```

Comment:
Not sure if this was carried over from another unit test, but if there's no reason to specify a timeout time, let's not specify it.